make.jl

```diff
@@ -44,21 +44,28 @@ local_path = @__DIR__
 ## The place to put the files from source_path after they go through Documenter.jl
 build_path = joinpath(tmp_path, "_docs")
 
-# Get any files from Turing's directory
-for (root, dirs, files) in walkdir(docs_path)
-    new_root = replace(root, docs_path => tmp_path)
-
-    for file in files
-        old_file = joinpath(root, file)
-        new_file = joinpath(new_root, file)
-        @debug "" old_file new_file
-        if !isdir(dirname(new_file))
-            mkpath(dirname(new_file))
+## The place to put the files for the local build
+build_path_local = joinpath(local_path, "_docs")
+
+# Get any files from Turing's directory for both live and local build
+for target_path in [tmp_path, build_path_local]
+    for (root, dirs, files) in walkdir(docs_path)
+        new_root = replace(root, docs_path => target_path)
+
+        for file in files
+            old_file = joinpath(root, file)
+            new_file = joinpath(new_root, file)
+            @debug "" old_file new_file
+            if !isdir(dirname(new_file))
+                mkpath(dirname(new_file))
+            end
+            cp(old_file, new_file, force=true)
         end
-        cp(old_file, new_file)
```

Comment:
IMO it would be easier to not duplicate the build (both since the code is a bit complicated/confusing and since now it is built twice) but only run the deploy parts conditionally on some CI environment variables, and only copy to the subdirectory when run locally. IIRC Documenter does something similar to disable deployments to Github when building locally.